Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```diff
@@ -256,3 +256,50 @@
   - type: Appearance
     visuals:
     - type: BarrelBoltVisualizer
+
+- type: entity
+  name: flare gun
+  parent: LauncherBase
+  id: FlareGun
+  description: A compact, single-shot pistol that fires shotgun shells.
```

Comment:
..is that what a flare gun is

Reply:
Yeah, you can actually load a flare gun with shotgun shells, but irl if the flare gun will probably explode if it's made out of plastic. You can probably find video on youtube.